packages/react/src/components/_InputWrapper/InputWrapper.tsx

```diff
@@ -104,12 +104,15 @@ export const InputWrapper = ({
     : undefined;
   const currentInputValue = value ? value.toString() : '';
 
+  const isFieldValid =
+    isValid ||
```

Comment:
just as i clicked approve I realised this is not correctly overriding the character limit check?

Reply:
if someone sends in `isValid={false}` the characterlimit check will still override this to true... I think we need to check if isValid is defined or not.